### PR TITLE
Added env variables for new organization and tests from main e2e

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: tests/neptune_fastai/test_e2e.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                                              item_tfms=Resize(224)
+                                              )
+
+        learn = cnn_learner(dls,
+                            squeezenet1_0,
+                            metrics=error_rate,
+                            cbs=[NeptuneCallback(run, 'experiment')],
+                            pretrained=False)
+
+        learn.fit(1)
```
Comment (shnela) [inline tests/neptune_fastai/test_e2e.py:56]: Why haven't you coppied following?
```
        learn.fit_one_cycle(1)
        learn.fine_tune(2)
```

File: tests/conftest.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
 @pytest.fixture()
 def run():
-    exp = neptune.init(mode='debug')
+    exp = neptune.init()
```
Comment (shnela) [inline tests/conftest.py:31]: Really this doesn't affect `test_base`?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -34,7 +43,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: 3.8
```
Comment (shnela) [inline .github/workflows/ci.yml:46]: 3.7 would be safer ;)

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -1,29 +1,38 @@
...
+      - name: Lint
+        run: |
+          python -m pylint -j 0 -f parseable neptune_fastai tests
   test:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # Windows builds fail because of the file header check - not clear what causes this.
-        # There are no pylint-file-header issues about this, and Windows builds succeed in the old domain
-        os: [ubuntu-latest, macos-latest] #, windows-latest]
```
Comment (shnela) [inline .github/workflows/ci.yml:10]: Why have you removed this comment?

**Review by shnela — CHANGES_REQUESTED** [review]

File: tests/neptune_fastai/test_e2e.py (hunk 1 above)
Comment (Raalsky) [reply tests/neptune_fastai/test_e2e.py:56]: For performance + it wasn't tested

File: .github/workflows/ci.yml (hunk 4 above)
Comment (Raalsky) [reply .github/workflows/ci.yml:10]: It was only reasonable for linting (which was moved to additional job that runs only on ubuntu)

File: .github/workflows/ci.yml (hunk 3 above)
Comment (Raalsky) [reply .github/workflows/ci.yml:46]: Or maybe I'll leave at 3.6

File: tests/conftest.py (hunk 2 above)
Comment (Raalsky) [reply tests/conftest.py:31]: ?

File: .github/workflows/ci.yml (hunk 4 above)
Comment (shnela) [reply .github/workflows/ci.yml:10]: But don't we want to test on windows?

File: tests/conftest.py (hunk 2 above)
Comment (shnela) [reply tests/conftest.py:31]: This fixture is not only used in e2e tests, but on the other hand it's working well, so no problem.

File: .github/workflows/ci.yml (hunk 4 above)
Comment (Raalsky) [reply .github/workflows/ci.yml:10]: Disabled as it failures on Windows due to lack of some additional steps (dataset download)

**Review by shnela — APPROVED** [review]

**Pull request merged**